x/bank/module.go

```diff
@@ -229,6 +231,7 @@ type bankInputs struct {
 	Cdc           codec.Codec
 	Subspace      paramtypes.Subspace
 	Key           *store.KVStoreKey
+	Authority     string
```

Comment:
I can't figure out what else is needed for this field to work. I'm assuming there will need to be a `` `key:...` `` tag, but I don't know how to actually set the value to inject.

Reply:
I gave up on trying to get this injected and instead just left it hard-coded in `provideModule`. I put a TODO over it to allow for injecting the value once that functionality is available.